plugins/hazelcast37/src/main/java/org/radargun/service/Hazelcast37ClientService.java

```diff
@@ -0,0 +1,57 @@
+package org.radargun.service;
+
+import com.hazelcast.client.HazelcastClient;
+import com.hazelcast.client.config.ClientConfig;
+import com.hazelcast.core.MemberAttributeEvent;
+import com.hazelcast.core.MembershipEvent;
+import com.hazelcast.core.MembershipListener;
+import org.radargun.Service;
+import org.radargun.config.Property;
+import org.radargun.traits.ProvidesTrait;
+
+/**
+ * Hazelcast client service
+ *
+ * @author Roman Macor &lt;rmacor@redhat.com&gt;
+ */
+
+@Service(doc = "Hazelcast client")
+public class Hazelcast37ClientService extends Hazelcast36Service {
+
+   @Property(doc = "List of server addresses the clients should connect to, separated by semicolons (;).")
+   protected String servers;
```

Comment:
Make the property `List<String>` (or `String[]`) and provide appropriate property converter. And what's single element, is that IP:port? In that case it should not be `List<String>` but `List<Address>`.